.cirrus.yml

```diff
@@ -968,6 +1000,13 @@ buildah_bud_test_task:
     alias: buildah_bud_test
     # Docs: ./contrib/cirrus/CIModes.md
     only_if: *not_tag_magic
+    # skip: - when PR (we never want to skip on nightly tests)
+    #       - when no danger files are changed
+    #       - when build source files are changed
+    skip: >-
+        $CIRRUS_PR != '' &&
+        !changesInclude('Makefile', 'contrib/cirrus/**', 'vendor/**', 'hack/**') &&
+        !changesInclude('**/*build*.go')
```

Comment:
This is making me more and more uncomfortable as the day progresses. I think this is >95% safe, but I can also envision situations where changes to storage code (podman, not vendor) or `CONF` envariable handling could impact `podman build`. Also, `bud.bats` runs podman commands other than `build`. I don't want to block, but I'd really like a lot of thought going into this.

Reply:
I cannot think of much, first of all consider that all podman build tests part of e2e/system tests are still run. This is only the buildah bud specific test that I skip here because it is both flaky and slow.
I honestly don't see much risk here and well that is why we run everything nightly. So at least the next day we should know that something broke an PR the last day (easy enough to check manually, or git bisect if needed).

IMO the benefit of less flakes faster CI outweighs the risk here.

Reply:
You are assuming that nightly cron runs are monitored. They aren't, not really. They flake so much that it's a cry-wolf problem: we just blindly press rerun but forget to check results. Sometimes we (I) don't even bother looking at failures unless they keep happening for multiple days in a row.